examples/chip-tool/templates/tests/partials/checks/maybeCheckExpectedConstraints.zapt

```diff
@@ -0,0 +1,36 @@
+{{#if hasExpectedConstraints}}
+    {{#if isOptional}}
+        VerifyOrReturn(CheckValuePresent("{{asPropertyValue forceNotOptional=true}}", {{asPropertyValue forceNotOptional=true}}));
```

Comment:
The `forceNotOptional` naming does not make sense here.   Should it be called `stopAtOptional` or `dontUnwrapValue` or something along those lines?

Reply:
I end up using `dontUnwrapValue`.